proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -655,7 +656,12 @@ message AuthenticateWebDeviceRequest {
   // Device web token to be spent in exchange for the device authentication
   // attempt.
   teleport.devicetrust.v1.DeviceWebToken device_web_token = 1;
+  string cluster_uri = 2;
```

Comment:
This was just missing from the initial iteration, but it only dawned on me when I got to writing the RPC. I don't want to do a breaking change, so tag number 2 it is.

Reply:
Is it possible to have the URI of a leaf cluster here? I believe it's always a root cluster, so the field should be named `root_cluster_uri`.

Reply:
Good call, done.